src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -65,7 +65,7 @@ public void CanGetGeneralInfoViaAPI()
             try
             {
                 var fullNode = this.apiTestsFixture.stratisPowNode.FullNode;
-                var ApiURI = fullNode.Settings.ApiUri;
+                var ApiURI = fullNode.NodeService<ApiSettings>().ApiUri;
```

Comment:
`apiUri` i believe is the correct name per our coding style (also many times below)

Reply:
Fixed the case introduced by legacy commit 292a2c8c:

`var apiURI = fullNode.NodeService<ApiSettings>().ApiUri;`